server/dash/dashboards/overviewComb.py

```diff
@@ -0,0 +1,178 @@
+
+# Request Type by Source Bar Chart
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=source_name&filter=created_date>=2016-01-01"
+df6 = pd.read_json(API_HOST + query_string)
+df6 = df6.groupby(['source_name'])['counts'].sum().to_frame()
+df6.sort_values('counts', ascending=False, inplace=True)
+df6.loc['Others'] = df6[4:].sum()
+df6.sort_values('counts', ascending=False, inplace=True)
+df6 = df6[:5]
+reqSourceBarchart = px.bar(
+    df6,
+    y=df6.index,
+    x='counts',
+    title="Total Requests by Source",
+    orientation='h'
+)
+
+# Median Request Days to Close Box Plot
+stas_df = pd.read_json(API_HOST + '/types/stats')
+stas_df = stas_df.sort_values('median', ascending=False)
+medDaysToCloseBoxPlot = go.Figure()
+medDaysToCloseBoxPlot.add_trace(
+    go.Box(
+        y=stas_df.type_name,
+        q1=stas_df['q1'],
+        median=stas_df['median'],
+        q3=stas_df['q3'],
+        marker_color='#29404F',
+        fillcolor='#E17C05',
+    )
+)
+medDaysToCloseBoxPlot.update_xaxes(
+
+    dtick=5
+)
+medDaysToCloseBoxPlot.update_layout(
+    title="Total Median Days to Close by Type",
+)
+
+# Day of Week Bar Chart:
+start_date = datetime.date.today() - datetime.timedelta(days=30)
+end_date = datetime.date.today() - datetime.timedelta(days=1)
+query_string = f"/reports?filter=created_date>={start_date}&filter=created_date<={end_date}"  # noqa
+print(" * Downloading data for dataframe")
+df = pd.read_json(API_HOST + query_string)
+df['created_date'] = pd.to_datetime(df['created_date'])
+dow_df = df.groupby(['created_date']).agg('sum').reset_index()
+dow_df['day_of_week'] = dow_df['created_date'].dt.day_name()
+numReqByDayOfWeekBarChart = px.bar(
+    dow_df,
+    x="day_of_week",
+    y="counts",
+    labels=LABELS,
+)
+numReqByDayOfWeekBarChart.update_xaxes(categoryorder='array', categoryarray= ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"])
+
+# Total Request by NC
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=council_name&filter=created_date>=2016-01-01"
+df1 = pd.read_json(API_HOST + query_string)
+df1 = df1.groupby(['council_name'])['counts'].sum().sort_values().to_frame()
+reqByNcBarChart = px.bar(
+    df1,
+    x=df1.index,
+    y='counts',
+    labels=LABELS,
+    title="Total Requests by Neighborhood Councils",
+)
+reqByNcBarChart.update_layout(font=dict(size=12))
+
+# LAYOUT
+layout = html.Div([
+    # Page 2 with Cards + Stuf
+    html.H1(title + " Pt. 1"),
+    html.P("The figures below represent the total number of 311 requests made across LA County from 2016-2021. In 2020, we saw an all-time high with more than 1.4 million requests.",
+           style={'font-size': '18px', 'font-style': 'italic'}),
+
+    html.Div([
+        html.Div([html.H2(f"{df2['counts'].sum():,}"), html.Label(
+            "Total Requests")], style={"text-align": 'center', "border": "0.5px black solid", 'width': '18vw', 'display': 'inline-block'}),
```

Comment:
The `style` arg here is used repeatedly, so let's create a constant for it so that we don't need to type it out so much (and read it so much).

Reply:
Please do that if it applies to other literals as well.

Reply:
Added LAYOUT VARIABLES section to set variables literals!